docs/http_transport.md

```diff
@@ -0,0 +1,79 @@
+How to use SDK HTTP Transport
+========================================================
+
+The SDK concept is based on using HTTP transport mechanism to send requests.
+The HTTP Transport is decoupled from the SDK architecture and uses just for sending HTTP requests.
+
+This principle allows SDK the use a different mechanism and libraries to reach the destination.
```

Comment:
"...allows for the SDK to use..." (but also consider if this is not necessary if you adopt the above recommendation

Reply:
applied